iree/samples/simple_embedding/CMakeLists.txt

```diff
@@ -51,6 +66,28 @@ iree_cc_binary(
     iree::vm::bytecode_module
 )
 
+iree_cc_binary(
+  NAME
+    simple_embedding_static
+  SRCS
+    "device_static.c"
+    "simple_embedding.c"
+  DEPS
+    ::static_embedding_module
+    iree::base
+    iree::hal
+    iree::hal::local
+    iree::hal::local::loaders::static_library_loader
+    iree::hal::local::task_driver
+    iree::modules::hal
+    iree::task::api
+    iree::vm
+    iree::vm::bytecode_module
+    ${CMAKE_CURRENT_SOURCE_DIR}/static_embedding_module.o
```

Comment:
I feel like we shouldn't check in platform/compiler specific binaries in samples like `static_embedding_module.o`. We can include instructions for how to generate files and run samples with them, but this is in an awkward spot where it may work on some machines but definitely won't work on others.

Reply:
No longer relevant since demo dropped from this PR.